utils/build-script-impl

```diff
@@ -241,11 +241,11 @@ function set_deployment_target_based_options() {
                     cmark_cmake_options=(
                         -DCMAKE_C_FLAGS="$(cmark_c_flags $deployment_target)"
                         -DCMAKE_CXX_FLAGS="$(cmark_c_flags $deployment_target)"
-                        -DCMAKE_OSX_SYSROOT:PATH="$(xcrun --sdk $xcrun_sdk_name --show-sdk-path)"
+                        -DCMAKE_OSX_SYSROOT:PATH=$(printf %q "$(xcrun --sdk $xcrun_sdk_name --show-sdk-path)")
```

Comment:
These paths are also already quoted, I feel like the issue is in some other place.